job_application/tests.py

```diff
@@ -0,0 +1,57 @@
+import pytest
+from jobboard.models import Job
+from student.models import Student, EducationalInstitution
+from django.contrib.auth.models import User
+from job_application.models import Application
+import datetime
+from django.db.models import QuerySet
+
+
+# The tests use the data that is already in the db from the migration files
+@pytest.fixture
+def institution_a():
+    institution_a = EducationalInstitution(name="A")
+    institution_a.save()
+    return institution_a
+
+
+@pytest.fixture
+def student_a(institution_a):
+    user_a = User.objects.create_user("test", "testpassword")
+    user_a.save()
+    student = Student(user=user_a, full_name="test user", email="test@test.com",
+                      date_of_birth=datetime.date(2020, 7, 13),
+                      phone_number="0000000000", educational_institution=institution_a,
+                      graduation_date=datetime.date(2020, 7, 13))
+    student.save()
+    return student
+
+
+@pytest.fixture
+def job_application(student_a):
+    job_a = Job.objects.get(title="job_a")
+    application_a = Application(
+        job=job_a, student=student_a, date_applied=datetime.date(2021, 7, 15))
+    application_a.save()
+    return application_a
+
+
+@pytest.mark.django_db
+def test_get_all_application_by_a_specified_job_returns_all_applications_as_a_query_set(job_application, student_a):
+    job_a = Job.objects.get(title="job_a")
+    application_temp_1 = Application.get_Applications_by_job(job_a)
+    assert isinstance(application_temp_1, QuerySet)
+    assert all(isinstance(application, Application) for application in application_temp_1)
+    assert list(application_temp_1.values_list("job", "student", "date_applied")) == [
+        (job_a.id, student_a.user.id, datetime.date(2021, 7, 15))]
+
+
+@pytest.mark.django_db
+def test_get_all_application_by_a_specified_student_returns_all_applications_as_a_query_set(job_application, student_a):
+    job_a = Job.objects.get(title="job_a")
+    application_temp_2 = Application.get_Applications_by_student(student_a)
+    print(application_temp_2)
```

Comment:
why do you print this?

Reply:
that just some printing I did to check something and that was left over by mistake. will be fixed.

Reply:
It's still there :)

Reply:
yeah, I don't know how I missed it. just delete it again now.

Reply:
and yet it's still there. maybe you forgot to do git add/commit?